src/content/docs/api/protocols/websockets.mdx

```diff
@@ -422,7 +430,7 @@ Message: {
     "type": "auxtimer",
     "payload": {
         "1": {
-            "duration": "count-up"
+            "duration": "count-down"
```

Comment:
:D 

Good catch!
In future, lets make these changes separate so they can be merged straight away without waiting for us to fiddle with documentation